apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/SessionPool.DatabaseStatistics.cs

```diff
@@ -29,6 +29,16 @@ public sealed class DatabaseStatistics
             /// </summary>
             public DatabaseName DatabaseName { get; }
 
+            /// <summary>
+            /// Database Role of sessions this set of statistics corresponds to. Can be null.
+            /// </summary>
+            public string DatabaseRole { get; }
+
+           /// <summary>
+           /// Partition Key of sessions this set of statistics corresponds to.
+           /// </summary>
+            private PartitionKey partitionKey { get; }
```

Comment:
If this is a property, name with PascalCase event if it's private. If it's an internal variable, name with _camelCase.